qemu/pcie.py

```diff
@@ -520,7 +521,7 @@ def fill_source(self):
             code = { "function": self.register_types }
         )
         self.source.add_type(
-            Type["type_init"].gen_usage(type_init_usage_init)
+            MacroType(Type["type_init"], initializer = type_init_usage_init)
```

Comment:
just rename `gen_usage` -> `gen_type`. You can use `MacroType` if you wish, but it results in one more `import`.

Reply:
No, better add `gen_type` replacing `MacroType` at all